app/routers/audio.py

```diff
@@ -0,0 +1,376 @@
+
+    Args:
+        is_audio_on (bool): True if the user chose to enable, False otherwise.
+        vol (Optional[int]): a number in the range (0, 1),
+        indicating the volume.
+        example: 0.4 means 40% of the tracks' volume.
+
+    Returns:
+        Optional[int]: returns the normalized volume, or None if audio
+        is disabled.
+    """
+    if is_audio_on:
+        vol /= 100
+
+    return vol
+
+
+# Functions for saving users' choices in the db.
+
+
+def save_audio_settings(
+    session: Session,
+    new_user: User,
+    music_choices: Optional[List[str]],
+    sfx_choice: Optional[str],
+        user_choices: Dict[str, Union[str, int]]):
+    """Save audio settings in the db.
+
+    Args:
+        session (Session): the database
+        new_user (User): default user. need to be replaced with real
+        one after registration is implemented
+        music_choices (Optional[List[str]]): a list of music tracks
+        f music is enabled, None otherwise.
+        sfx_choice (Optional[str]): choice for sound effect.
+        user_choices (Dict[str, Union[str, int]]):
+        including music_on, music_vol, sfxs_on, sfxs_vol
+    """
+    # need to change later to get the real user,
+    # when registration feature is ready.
+    user_name = "new_user"
+    user = get_user(session, user_name, new_user)
+    user_id = user.id
+
+    handle_audio_settings(session, user, user_choices)
+    handle_user_audio_tracks(session, user_id, music_choices, sfx_choice)
+
+
+def get_user(session: Session, user_name: str, new_user: User) -> User:
+    """Retrieves user according to user_name.
+    if he doesn't exist - returns default user (new_user).
+
+    Args:
+        session (Session): the database.
+        user_name (str): name of the user.
+        new_user (User): defualt user for the case that the
+        user with user_name isn't found in the database.
+
+    Returns:
+        User: a user object.
+    """
+    user = session.query(User).filter_by(username=user_name).first()
+    if not user:
+        session.add(new_user)
+        session.commit()
+        user = session.query(User).filter_by(
+            username=new_user.username).first()
+    return user
+
+
+def handle_audio_settings(
+    session: Session,
+    user: User,
+        user_choices: Dict[str, Union[str, int]]):
+    """Insert or update a new record into AudioSettings table.
+    The table stores the following information:
+    music on, music_vol, sfxs on, sfxs_vol.
+
+    Args:
+        session (Session): the database
+        user (User): current user.
+        user_choices (Dict[str, Union[str, int]]):
+        including music_on, music_vol, sfxs_on, sfxs_vol
+    """
+    user_id = user.id
+    audio_settings = session.query(AudioSettings).filter_by(
+        user_id=user_id).first()
+    if not audio_settings:
+        audio_settings = AudioSettings(user_id=user_id, **user_choices)
+        session.add(audio_settings)
+        session.commit()
```

Comment:
I think you can use [`merge`](https://docs.sqlalchemy.org/en/13/orm/session_api.html#sqlalchemy.orm.session.Session.merge) here. Either way, prefer putting `.commit` outside the `if-else` clauses (to prevent DRY code)